kit/src/main/java/com/oracle/javafx/scenebuilder/kit/metadata/Metadata.java

```diff
@@ -4470,7 +4472,12 @@ public boolean isPropertyTrimmingNeeded(PropertyName name) {
                 com.oracle.javafx.scenebuilder.kit.metadata.property.value.DoublePropertyMetadata.DoubleKind.COORDINATE,
                 true, /* readWrite */
                 100.0, /* defaultValue */
-                new InspectorPath("Properties", "Specific", 98));
+                new InspectorPath("Properties", "Specific", 10));
+    private final ValuePropertyMetadata useSystemMenuBarNamePropertyMetadata =
```

Comment:
Rename `useSystemMenuBarNamePropertyMetadata` to `useSystemMenuBarPropertyMetadata`